test/unit/vdom/events.js

```diff
@@ -29,31 +26,11 @@ describe('vdom/events (on*)', () => {
     fixture(el);
 
     afterMutations(
-      () => {}, // .render()
-      () => {
-        const shadowDiv = el.shadowRoot.children[0];
-
-        // Ensures that it rendered.
-        expect(shadowDiv.textContent).to.equal('0');
-        expect(el._test).to.equal(0);
-
-        // Trigger the handler.
-        emit(shadowDiv, 'event');
-
-        // Ensure the event fired.
-        expect(el._test).to.equal(1);
-
-        // Re-render.
-        props(el, { test: el.test + 1 });
-        expect(shadowDiv.textContent).to.equal('1');
-        emit(shadowDiv, 'event');
-        expect(el._test).to.equal(2);
-
-        props(el, { test: el.test + 1 });
-        expect(shadowDiv.textContent).to.equal('2');
-        emit(shadowDiv, 'event');
-        expect(el._test).to.equal(3);
-      },
+      () => expect(el.shadowRoot.textContent).to.equal('0'),
```

Comment:
This essentially tests the same thing in a simpler manner. However the previous test didn't take into account things that may happen after mutations very well, this ensures that anything that might happen in a microtask, should happen, and that things are more direclty tested.